presage/src/manager.rs

```diff
@@ -953,6 +959,65 @@ impl<C: Store> Manager<C, Registered> {
                                         log::trace!("{group:?}");
                                     }
                                 }
+                                let uuid = content.metadata.sender.uuid.clone();
+                                match state.config_store.contact_by_id(uuid) {
+                                    Ok(Some(_)) => {}
+                                    Ok(None) => {
+                                        // Create a new contact
+                                        if let ContentBody::DataMessage(DataMessage {
+                                            profile_key: Some(profile_key),
+                                            ..
+                                        }) = &content.body
+                                        {
+                                            let contact = Contact {
+                                                uuid: uuid.clone(),
+                                                name: "".to_string(),
+                                                color: None,
+                                                phone_number: None,
+                                                verified: Verified {
+                                                    identity_key: None,
+                                                    state: None,
+                                                    destination_uuid: None,
+                                                    null_message: None,
+                                                },
+                                                blocked: false,
+                                                expire_timer: 0,
+                                                inbox_position: 0,
+                                                archived: false,
+                                                avatar: None,
+                                                profile_key: profile_key.to_vec(),
+                                            };
```

Comment:
I guess most of the fields here are default values to you could try to only set `uuid` and add at the end

```
..Default::default()
```

Reply:
the trait `std::default::Default` is not implemented for `libsignal_service::models::Contact`.